src/components/Table/Table.tsx

```diff
@@ -174,39 +188,105 @@ class Table extends React.Component<Props, State> {
 
   // Function to render tbody & td with specifc data & if user passed any custom component that can also get rendered
   renderBody = () => {
-    const { column, rowAction, selectRow } = this.props;
-    const { data } = this.state;
+    const { children, column, hideRow, selectRow } = this.props;
+    const { data, expandedRow } = this.state;
+
+    if (!children) {
+      return (
+        <TableBody>
+          {
+            data.map((item: any, index: number) => {
+              if (!this.props.hideRow || !this.hideRow(item)) {
```

Comment:
this.hideRow, where is it defined??

Reply:
@ankitsaraogi its defined in this component itself, line number 139